apps/site/assets/css/_recently-visited.scss

```diff
@@ -1,8 +1,4 @@
 .c-recently-visited {
-  h2 {
-    margin-top: 0;
```

Comment:
This is used on a few other pages too - are we OK with changing those?

Reply:
I think this was originally introduced as part of the homepage design updates, so this change should effectively revert any other pages whilst keeping the homepage updates intact (which I believe was the point of the ticket)

Reply:
oh ha! Awesome then